internal/service/autoscaling/policy.go

```diff
@@ -87,6 +87,276 @@ func ResourcePolicy() *schema.Resource {
 							MaxItems: 1,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									"customized_scaling_metric_specification": {
+										Type:          schema.TypeList,
+										Optional:      true,
+										MaxItems:      1,
+										ConflictsWith: []string{"predictive_scaling_configuration.0.metric_specification.0.predefined_scaling_metric_specification"},
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"metric_data_queries": {
+													Type:     schema.TypeList,
+													Required: true,
+													MaxItems: 10,
+													Elem: &schema.Resource{
+														Schema: map[string]*schema.Schema{
+															"id": {
```

Comment:
Small nit: can we order the arguments alphabetically? This is also done at the AWS SDK Go documentation: https://docs.aws.amazon.com/sdk-for-go/api/service/autoscaling/#MetricDataQuery would make it easier to read when having the documentation at the side

Reply:
Sure, will update.